### PR TITLE
export `call-pstn-full` and `sfb-viewbox` icons

### PR DESCRIPTION
**Pull request by thewulf7**
## Description

This PR exports `call-pstn-full` and `sfb-viewbox` icons

### CODE REVIEW

**Comment by kuzhelov** [comment]
changes look fine to me - @codepretty, could you, please, take final look on the names of the exported  icons?

File: CHANGELOG.md
Comment (kuzhelov) [inline CHANGELOG.md]: nit: please, lowercase `Theme` -> `theme`

Also, please, ensure that changelog entry is provided under `Unreleased` section after merging with recent `master`.

**Comment by kuzhelov** [comment]
@thewulf7, icons were checked against spec, there is nothing blocking this PR - lets merge recent master into this branch, resolve CHANGELOG conflicts and merge right after 👍

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-pstn-full.tsx
Comment (kuzhelov) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-pstn-full.tsx:11]: 👍

**Review by kuzhelov — APPROVED** [review]

**Review by bmdalex — APPROVED** [review]

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-sfb-viewbox-32.tsx
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-sfb-viewbox-32.tsx]: Can we name the exported icon 'skype' or 'skype-logo'?

File: packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx]: this should be `teamsIconClassNames.outline`

File: packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx:14]: this should be `className={cx(teamsIconClassNames.filled, classes.filledPart)`

File: packages/react/src/themes/teams/components/Icon/svg/icons/sfbViewbox.tsx
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/sfbViewbox.tsx]: i believe the `aria-labelledby` can be removed

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-pstn.tsx
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-pstn.tsx:14]: Can we rename this to something more generic that describes what the icon looks like 'call-user'?

Comment (codepretty) [reply packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-pstn.tsx:14]: Since we're merging 2 icons into 1, can you also add an `exportedAs` to `icons-call-pstn-full` as well

Comment (thewulf7) [reply packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-pstn.tsx:14]: PSTN is public switched telephone network. It differs from user semantically, as its subset of all users.

Comment (thewulf7) [reply packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-pstn.tsx:14]: And exportedAs already added to `icons-call-pstn-full`

File: packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx
Comment (thewulf7) [reply packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx:14]: sorry but why?

File: packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx
Comment (thewulf7) [reply packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx]: sorry but why?

File: packages/react/src/themes/teams/components/Icon/svg/icons/sfbViewbox.tsx
Comment (thewulf7) [reply packages/react/src/themes/teams/components/Icon/svg/icons/sfbViewbox.tsx]: fixed

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-sfb-viewbox-32.tsx
Comment (thewulf7) [reply packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-sfb-viewbox-32.tsx]: fixed

**Review by codepretty — APPROVED** [review]

File: packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx
Comment (codepretty) [reply packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx:14]: because this path of the svg is the filled state and if the classes aren't correct then it won't show up correctly

File: packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx
Comment (codepretty) [reply packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx]: this is needed to add the correct static class for outline parts of the svg

File: packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx
Comment (mnajdova) [inline packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx]: change this with relative import

Comment (thewulf7) [reply packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx]: fixed

**Pull request merged**
